lib/exceptional_synchrony/event_machine_proxy.rb

```diff
@@ -42,7 +42,9 @@ def sleep(seconds)
     end
 
     def yield_to_reactor
-      @synchrony.sleep(0)
+      if reactor_running?
+        @synchrony.sleep(0)
```

Comment:
It does. This change will allow the PNAPI initial load of the dnp to happen before we've started the reactor (after a minor refactor on the PNAPI side).